src/test/test-execute.c

```diff
@@ -154,6 +155,19 @@ static void test_exec_runtimedirectory(Manager *m) {
                 test(m, "exec-runtimedirectory-owner.service", 0, CLD_EXITED);
 }
 
+static void test_exec_capabilityboundingset(Manager *m) {
+        /* We use capsh to test if the capabilities are
+         * properly set, so be sure that it exists */
+        if (find_binary("capsh", NULL) != 0)
+                return;
```

Comment:
On Thu, Oct 29, 2015 at 4:07 PM, Evgeny Vereshchagin <
notifications@github.com> wrote:

> In src/test/test-execute.c
> <https://github.com/systemd/systemd/pull/1714#discussion_r43396710>:
>
> > @@ -154,6 +155,19 @@ static void test_exec_runtimedirectory(Manager *m) {
> >                  test(m, "exec-runtimedirectory-owner.service", 0, CLD_EXITED);
> >  }
> >
> > +static void test_exec_capabilityboundingset(Manager *m) {
> > +        /* We use capsh to test if the capabilities are
> > +         * properly set, so be sure that it exists */
> > +        if (find_binary("capsh", NULL) != 0)
> > +                return;
>
> Is it silently skip test?
>

Ok, I will add a log for this and also where we test if we have the
user/group nobody.


> Maybe, log_*?
> Something like that
> <https://github.com/systemd/systemd/blob/master/src/test/test-capability.c#L73>
> :
>
>         if (!nobody) {
>                 log_error_errno(errno, "Could not find nobody user: %m");
>                 return -EXIT_TEST_SKIP;
>         }
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/systemd/systemd/pull/1714/files#r43396710>.
>